scripts/main.js

```diff
@@ -1,484 +1,22 @@
-        this.cursor.remove();
-      }
-			this.strPos = 0;
-			this.arrayPos = 0;
-			this.curLoop = 0;
-			// Send the callback
-			this.options.resetCallback();
-		}
-
-	};
-
-	$.fn.typed = function(option) {
-		return this.each(function() {
-			var $this = $(this),
-				data = $this.data('typed'),
-				options = typeof option == 'object' && option;
-			if (data) { data.reset(); }
-			$this.data('typed', (data = new Typed(this, options)));
-			if (typeof option == 'string') data[option]();
-		});
-	};
-
-	$.fn.typed.defaults = {
-		strings: ["These are the default values...", "You know what you should do?", "Use your own!", "Have a great day!"],
-		stringsElement: null,
-		// typing speed
-		typeSpeed: 0,
-		// time before typing starts
-		startDelay: 0,
-		// backspacing speed
-		backSpeed: 0,
-		// shuffle the strings
-		shuffle: false,
-		// time before backspacing
-		backDelay: 500,
-		// loop
-		loop: false,
-		// false = infinite
-		loopCount: false,
-		// show cursor
-		showCursor: true,
-		// character for cursor
-		cursorChar: "|",
-		// attribute to type (null == text)
-		attr: null,
-		// either html or text
-		contentType: 'html',
-		// call when done callback function
-		callback: function() {},
-		// starting callback function before each string
-		preStringTyped: function() {},
-		//callback for every typed string
-		onStringTyped: function() {},
-		// callback for reset
-		resetCallback: function() {}
-	};
-
-
-}(window.jQuery);
-
-
-$(document).ready(function($) {
-    $(".typed").typed({
-        strings: ["student", "associate consultant"],
-        typeSpeed: 50,
-        backSpeed: 0,
-        shuffel: true,
-        loop: true,
-        backDelay: 1000
-    });
-
-    var $carousel;
-    var $cellElems = $(".slideshowCell");
-    $(".postImgContainer").on('click', function(event) {
-        event.preventDefault();
-        var slideshowAttr = $(this).attr('slideshow');
-        var slideshowImages = slideshowAttr.split(",");
-        var path = $(this).attr('slideshowpath');
-
-        $(".slideshowOverlay").show();
-        $carousel = $(".slideshow").flickity({
-            wrapAround: true,
-            imagesLoaded: true,
-            pageDots: false,
-            lazyLoad: 3
-        });
-        $(slideshowImages).each(function(index, el) {
-            $cellElems = $("<div class='slideshowCell'><img class='slideshowCellImage' data-flickity-lazyload='" + path + el + "' /></div>");
-            $carousel.flickity( 'append', $cellElems );
-        });
-
-    });
-
-    $(".closeBtn").on('click', function(event) {
-        event.preventDefault();
-        $(".slideshowOverlay").hide();
-        $(".slideshow").flickity('destroy');
-        $(".slideshow").empty();
-
-    });
-
-    window.sr = ScrollReveal({ reset: true });
-    sr.reveal('.post');
-
-});
+const ityped = require('ityped')
+const tiltjs = require('vanilla-tilt')
+const ScrollReveal = require('scrollreveal')
+
```

Comment:
Using CDN in head.html for now since there aren't any good npm versions. May create my own in the future.